kafka-ui-react-app/src/lib/hooks/api/topics.ts

```diff
@@ -120,14 +120,15 @@ export function useCreateTopicMutation(clusterName: ClusterName) {
         clusterName,
         topicCreation: formatTopicCreation(data),
       }),
-    {
-      onSuccess: () => {
-        showSuccessAlert({
-          message: `Topic successfully created.`,
-        });
-        client.invalidateQueries(topicKeys.all(clusterName));
```

Comment:
`client.invalidateQueries(topicKeys.all(clusterName));`
this has to be left intact I suppose
Also, please remove the code instead of commenting it